src/helper.ts

```diff
@@ -0,0 +1,16 @@
+export const arrayContainsEntries = <T>(collection: T[]) =>
+  Array.isArray(collection) && collection.length > 0;
+
+// TODO: add unit tests
+// TODO: Better naming for this?
+// Cover undefined and null cases, returns 0 in that case
+export const safeArrayLength = <T>(collection: T[]) =>
```

Comment:
This is a private method isn't it?

Reply:
I think it can be a generic goodie that could be used in other places